cli/src/commands/next.rs

```diff
@@ -65,6 +65,9 @@ pub(crate) struct NextArgs {
     /// edit`).
     #[arg(long, short)]
     edit: bool,
+    /// Jump to the next conflicted descendant.
+    #[arg(long, conflicts_with = "offset")]
+    conflict: bool,
```

Comment:
I think you want to either make `offset` and `conflict` into a group so you cannot specify both, or you want it to move to the nth conflict.

Reply:
I think this is good enough at the moment, as it currently only is a single option.